src/System.IO.FileSystem/tests/File/GetSetTimes.cs

```diff
@@ -107,5 +107,20 @@ public void SetLastWriteTimeMilliSec()
             long secondFileTicks = File.GetLastWriteTimeUtc(secondFile).Ticks;
             Assert.True(firstFileTicks <= secondFileTicks, $"First File Ticks\t{firstFileTicks}\nSecond File Ticks\t{secondFileTicks}");
         }
+
+        [Fact]
+        public void SetLastAccessTimeTicks()
+        {
+            string firstFile = GetTestFilePath();
```

Comment:
If we were always reading and woriting zero milliseconds on some OS, these tests would still pass, right? Why not use the loop and check pattern from my original tests  to make sure milliseconds are non zero.

Reply:
When we create, move or copy the file, the granularity is upto milliseconds(HFS\)microseconds\nanoseconds(mostly this).
But when we set the lastaccess\write\createTime property, the granularity for that property is reduced to milliseconds. The lastaccess\lastwrite property are interlinked with each other. They are set together on the native side.
This test ensures that there is no loss of info for LastAccess if we set LastWrite property and vice versa.(the previous PR corrected this)